src/Libraries/Microsoft.Extensions.Compliance.Abstractions/Classification/DataClassificationSet.cs

```diff
@@ -83,7 +83,16 @@ public DataClassificationSet Union(DataClassificationSet other)
     /// Gets a hash code for the current object instance.
     /// </summary>
     /// <returns>The hash code value.</returns>
-    public override int GetHashCode() => _classifications.GetHashCode();
+    public override int GetHashCode()
```

Comment:
Probably goes without saying, but make sure to add some test coverage to ensure the fix works, and to prevent regressions.